applicationLibrary/molecularDynamics/molecularDynamicsLibrary/LJFunctor.h

```diff
@@ -684,6 +737,100 @@ class LJFunctor
+   * - accumulate force on mol j if n3: 3
+   * - Total: 15 without n3, 18 with n3
+   *
+   * For the globals calculation, this is:
+   * - virial: 3
+   * - potential: 1, or 2 with shift
+   * - accumulation: 4
+   * - multiplications by 0.5 and second accumulations are not treated as useful FLOPs, as these are just workarounds
+   *   to avoid accumulating globals on owned/halo borders twice
+   * - Total: 8, or 9 with shift
+   *
+   * Caveats:
+   *
+   * This function is supposed to return useful FLOPs, e.g. without counting masked vector instructions or
+   * workarounds for accumulating globals on owned/halo borders twice. You could argue that, strictly speaking, we
+   * redundantly calculate forces and globals twice in the newton3 case. This function does not treat such "redundant"
+   * calculations as useless.
+   *
+   * Similarly, this function does not treat halo-halo interactions as redundant useless calculations.
+   *
+   * @return number of FLOPs since initTraversal() is called.
+   */
+  size_t getNumFLOPs() {
+    if constexpr (countFLOPs) {
+      size_t numDistCallsAcc = std::accumulate(_aosThreadDataFLOPs.begin(), _aosThreadDataFLOPs.end(), 0ul,
+                                               [](size_t sum, const auto &data) { return sum + data.numDistCalls; });
+      size_t numKernelCallsN3Acc =
+          std::accumulate(_aosThreadDataFLOPs.begin(), _aosThreadDataFLOPs.end(), 0ul,
+                          [](size_t sum, const auto &data) { return sum + data.numKernelCallsN3; });
+      size_t numKernelCallsNoN3Acc =
+          std::accumulate(_aosThreadDataFLOPs.begin(), _aosThreadDataFLOPs.end(), 0ul,
+                          [](size_t sum, const auto &data) { return sum + data.numKernelCallsNoN3; });
+      size_t numGlobalCalcsAcc =
+          std::accumulate(_aosThreadDataFLOPs.begin(), _aosThreadDataFLOPs.end(), 0ul,
+                          [](size_t sum, const auto &data) { return sum + data.numGlobalCalcs; });
+
+      AutoPasLog(TRACE, "Number of Distance Calls           = {}", numDistCallsAcc);
+      AutoPasLog(TRACE, "Number of Newton3 Kernel Calls     = {}", numKernelCallsN3Acc);
+      AutoPasLog(TRACE, "Number of Non-Newton3 Kernel Calls = {}", numKernelCallsNoN3Acc);
+      AutoPasLog(TRACE, "Number of Global Calculations      = {}", numGlobalCalcsAcc);
+
+      constexpr size_t numFLOPsPerDistanceCall = 8;
+      constexpr size_t numFLOPsPerN3KernelCall = 18;
+      constexpr size_t numFLOPsPerNoN3KernelCall = 15;
+      constexpr size_t numFLOPsPerGlobalCalc = applyShift ? 9 : 8;
+
+      return numDistCallsAcc * numFLOPsPerDistanceCall + numKernelCallsN3Acc * numFLOPsPerN3KernelCall +
+             numKernelCallsNoN3Acc * numFLOPsPerNoN3KernelCall + numGlobalCalcsAcc * numFLOPsPerGlobalCalc;
+    } else {
+      autopas::utils::ExceptionHandler::exception("LJFunctor::getNumFLOPs called without countFLOPs enabled!");
+      return 0;
+    }
+  }
+
+  double getHitRate() {
```

Comment:
```suggestion
  double getHitRate() override {
```
Also missing for all other instantiations of this function.

Reply:
Done in 7fa81126